src/main/kotlin/com/mparticle/kits/UrbanAirshipKit.kt

```diff
@@ -206,7 +206,8 @@ class UrbanAirshipKit : KitIntegration(), KitIntegration.PushListener, KitIntegr
                 .apply()
         }
         if (identityType == configuration?.userIdField) {
-            UAirship.shared().namedUser.id = null
+            val ass =UAirship.shared().contact.editAttributes()
```

Comment:
Can you change this variable name?

Reply:
should it just be
```suggestion
            UAirship.shared().contact.editAttributes()
```
doesn't seem like the value is used

Reply:
This line is removed complet it meant to be uass for urbanAirShipShared, supoer sorry about that